client/src/components/Shared/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+
+const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
+    <Tooltip {...props} classes={{ popper: className }} />
+  ))(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+      backgroundColor: '#f5f5f9',
+      color: 'rgba(0, 0, 0, 0.87)',
+      maxWidth: 220,
+      minWidth: 160,
+      fontSize: theme.typography.pxToRem(12),
+      border: '1px solid #dadde9',
+    },
+  }));
+
+interface Props {
+    display_text: string;
+    hover_texts: any;
+}
+
+export const SourcesTooltip: React.FC<Props> = ({display_text,hover_texts}) => {
+    return (
+
+        <div>
+          <HtmlTooltip
+            title={
+              <React.Fragment>
```

Comment:
i prefer using the shorthand for fragments, like:
```suggestion
              <>
```